_troubleshooting/mst/transaction-wraparound.md

```diff
@@ -0,0 +1,33 @@
+---
+title: PostgreSQL transaction ID wraparound
+section: troubleshooting
+products: [mst]
+topics: [transaction ID]
```

Comment:
```suggestion
```

IMO 'transaction ID' isn't hefty enough to qualify as its own topic. I see topics as things like "hypertables", "schemas" that have their own section in how-to guides. Might be a bit nitpicky but this will help us keep things clean when we try to auto-generate pages.